storage/range_tree.go

```diff
@@ -25,37 +25,27 @@ import (
 )
 
 type (
```

Comment:
I think the rest of the code doesn't use type blocks like this. I don't feel strongly about it.